nidx/nidx_relation/src/lib.rs

```diff
@@ -120,4 +129,36 @@ impl RelationSearcher {
     pub fn search(&self, request: &RelationSearchRequest) -> anyhow::Result<RelationSearchResponse> {
         self.reader.search(request)
     }
+
+    pub fn suggest(&self, prefixes: Vec<String>) -> Vec<RelationNode> {
+        let requests =
+            prefixes.par_iter().filter(|prefix| prefix.len() >= MIN_SUGGEST_PREFIX_LENGTH).cloned().map(|prefix| {
```

Comment:
Adding rayon for this. Do we really want it?

Reply:
I think we can live without this and just use a plain iterator. If we find the need for speed here, we can add it later.

I don't think we will have many prefixes anyway (one per word on the query) and I'd rather remove the complexity until we are sure we need it. It's easy to add back again if we want to.